docs/pages/more/create-expo.mdx

```diff
@@ -0,0 +1,170 @@
+
+**If you are migrating from one package manager to another**, you've to manually carry out the additional configuration in your project. **If you are using [EAS](/eas/)**, you also have to configure your project for any additional required steps manually.
+
+All the additional steps for each package manager are listed below.
+
+### npm
+
+#### Local installation
+
+npm is installed as part of Node.js installation. See [Node.js documentation](https://nodejs.org/en/download/package-manager) for installation instructions.
+
+#### EAS installation
+
+Supported by default if the project directory contains **package-lock.json**.
+
+### Yarn 1 (Classic)
+
+#### Local installation
+
+Yarn 1 (Classic) is usually installed as a global dependency of npm. See [Yarn 1 documentation](https://classic.yarnpkg.com/en/docs/getting-started) for installation instructions.
+
+#### EAS installation
+
+Supported by default if the project directory contains **yarn.lock**.
+
+### Yarn 2+ (Modern)
+
+#### Local installation
+
+Yarn 2+ handles package management differently than Yarn 1. One of the core changes in Yarn 2+ is the PnP node linking model that does not work with React Native.
+
+See [Yarn documentation](https://yarnpkg.com/getting-started/install) for installation instructions.
+
+#### Additional configuration steps
+
+After installing Yarn Modern on your local development environment, you've to specify the type linker to install npm packages using [`nodeLinker`](https://yarnpkg.com/features/linkers#nodelinker-node-modules).
+
+Create **.yarnrc.yml** in your project's root directory with the following configuration:
+
+```yml .yarnrc.yml
+nodeLinker: node-modules
+```
+
+#### EAS installation
+
+Yarn Modern on EAS requires an [additional configuration step](#additional-configuration-steps) and adding [`eas-build-pre-install` hook](/build-reference/npm-hooks/) in your project's **package.json**, add the following configuration:
+
+```json package.json
+{
+  "scripts": {
+    "eas-build-pre-install": "corepack enable && yarn set version 4"
+  }
+}
+```
+
+### pnpm
+
+#### Local installation
+
+Requires installing Node.js. See [pnpm documentation](https://pnpm.io/installation) for installation instructions.
+
+#### Additional configuration steps
+
+After installing pnpm on your local development environment, you've to specify the type linker to install npm packages using [`node-linker`](https://pnpm.io/npmrc#node-linker).
+
+Create **.npmrc** in your project's root directory and add the following configuration:
+
+```ini .npmrc
+node-linker=hoisted
+```
+
+#### EAS installation
+
+Supported by default if the project directory contains **pnpm-lock.yaml** and the [additional configuration step](#additional-configuration-steps-1) followed.
+
+### Bun
+
+> **warning** Bun version `1.0.22` or below is supported due to [peer dependency conflicts](https://github.com/oven-sh/bun/issues/9135).
```

Comment:
```suggestion
> **warning** Only Bun versions `1.0.22` or below are supported due to [peer dependency conflicts](https://github.com/oven-sh/bun/issues/9135).
```

Let's clear the message if the "below" is not a typo, otherwise let's correct it.

Reply:
I'm going to remove this warning based on this message earlier this week: https://exponent-internal.slack.com/archives/C1QLM27T9/p1718034956190959?thread_ts=1718032071.445799&cid=C1QLM27T9

Reply:
That makes sense, thanks for the context! 👍